scripts/build.sh

```diff
@@ -45,17 +45,17 @@ if [[ ${CMD} == "build" && (-z ${GOOS} || -z ${GOARCH}) ]]; then
 fi
 
 FLAGS="\
-    -X 'main.Version=${VERSION}' \
-    -X 'main.GitCommit=${COMMIT}' \
-    -X 'main.BuildDate=${BUILD_DATE}' \
-    -X 'main.BuildOS=${BUILD_OS}'" \
+    -X 'github.com/anz-bank/sysl/pkg/command.Version=${VERSION}' \
+    -X 'github.com/anz-bank/sysl/pkg/command.GitCommit=${COMMIT}' \
+    -X 'github.com/anz-bank/sysl/pkg/command.BuildDate=${BUILD_DATE}' \
+    -X 'github.com/anz-bank/sysl/pkg/command.BuildOS=${BUILD_OS}'" \
```

Comment:
These should remain in `cmd/sysl`. Use some kind of API to pass the info into `pkg/command`.